package.json

```diff
@@ -67,7 +68,7 @@
 		"utf8": "^3.0.0"
 	},
 	"optionalDependencies": {
-		"encoding": "^0.1.12"
+		"fetch-charset-detection": "^0.2.0"
```

Comment:
I don't know if I can be more clear, but no, we don't want dependency on fetch-charset-detection at all, not even optional